src/app/pages/LandingPage/components/TopBanner/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { translations } from 'locales/i18n';
+
+// import { BannerPromotion } from '../BannerPromotion';
+import styles from './index.module.scss';
+
+export const TopBanner = () => {
+  const { t } = useTranslation();
+  return (
+    <div className={styles.wrapper}>
+      <div className={styles.sunPatternWrapper}>
+        <div className="tw-absolute tw-w-full tw-h-full tw-top-0 tw-left-0 tw-flex tw-flex-col tw-justify-center tw-text-center">
+          <p className={styles.welcomeTitle}>
+            {t(translations.landingPage.welcomeTitle)}
```

Comment:
![UI issue 3](https://user-images.githubusercontent.com/56412037/144448605-ff3265fd-d55e-4416-8cb0-a23a1a191658.JPG)
In UI/UX design, there are 4 cards which shows rewards, but there is no code according to that.
Image attached.

Reply:
This should have been updated in the design by @phpbro

To start with, we won't be having any reward program, but these will be added once we launch.